### PR TITLE
fix(hardware): changed float conversion, updated integration test values

### PR DESCRIPTION
**Pull request by caila-marashaj**

### CODE REVIEW

File: hardware/opentrons_hardware/sensors/utils.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def build(cls, data):  # type: ignore[no-untyped-def]
 
     def to_float(self) -> float:
         """Convert data to float."""
-        return (1.0 * self.as_int) / 2**15
+        return (1.0 * self.as_int) / 2**16
```
Comment (amitlissack) [inline hardware/opentrons_hardware/sensors/utils.py]: Perhaps `2**16` should be defined as a named constant in this file?

**Review by amitlissack — APPROVED** [review]
One little question

File: hardware/opentrons_hardware/sensors/utils.py (hunk 1 above)
Comment (amitlissack) [reply hardware/opentrons_hardware/sensors/utils.py]: Please replace all instances of `2**16` in this PR to `int_to_float`

File: hardware/opentrons_hardware/scripts/monitor_sensors.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ async def do_run(
     """Configure and start the monitoring."""
     threshold_payload = payloads.SetSensorThresholdRequestPayload(
         sensor=fields.SensorTypeField(constants.SensorType.capacitive),
-        threshold=Int32Field(int(threshold * 2**15)),
+        threshold=Int32Field(int(threshold * 2**16)),
```
Comment (amitlissack) [inline hardware/opentrons_hardware/scripts/monitor_sensors.py]: `int_to_float`

File: hardware/opentrons_hardware/scripts/sensor_pass.py [hunk 3]
```diff
@@ -136,7 +136,7 @@ async def run(args: argparse.Namespace) -> None:
 
     threshold_payload = payloads.SetSensorThresholdRequestPayload(
         sensor=fields.SensorTypeField(SensorType.capacitive),
-        threshold=Int32Field(int(args.threshold * 2**15)),
+        threshold=Int32Field(int(args.threshold * 2**16)),
```
Comment (amitlissack) [inline hardware/opentrons_hardware/scripts/sensor_pass.py]: `int_to_float`

File: hardware/opentrons_hardware/sensors/utils.py [hunk 4]
```diff
@@ -5,6 +5,8 @@
 from opentrons_hardware.firmware_bindings.constants import NodeId, SensorType
 from opentrons_hardware.firmware_bindings.utils.binary_serializable import Int32Field
 
+int_to_float = 2**16
```
Comment (amitlissack) [inline hardware/opentrons_hardware/sensors/utils.py]: Is `int_to_float` the right name for this? Should it be something like `max_threshold_int` or something?

Can we mark it as `Final` also?

Comment (caila-marashaj) [reply hardware/opentrons_hardware/sensors/utils.py]: yeah I just wasn't sure what to name it to be honest- that sounds good to me. And yes I will mark it as `Final`

Comment (caila-marashaj) [reply hardware/opentrons_hardware/sensors/utils.py]: Actually I believe the version of Python we're on doesn't have Final- I added it and pushed that, and the lint check on GitHub didn't like it

Comment (Laura-Danielle) [reply hardware/opentrons_hardware/sensors/utils.py]: I think the better name would be `sensor_fixed_point_conversion`

**Review by Laura-Danielle — APPROVED** [review]
Looks good in general, just please add in the `Final` keyword and change `max_threshold_int` to `sensor_fixed_point_conversion`.

File: hardware/opentrons_hardware/sensors/utils.py (hunk 4 above)
Comment (amitlissack) [reply hardware/opentrons_hardware/sensors/utils.py]: You're right @caila-marashaj you have to do ```from typing_extensions import Final```

**Pull request merged**